core/src/main/java/org/bitcoinj/quorums/FinalCommitment.java

```diff
@@ -55,6 +57,11 @@ protected void parse() throws ProtocolException {
 
         llmqType = readBytes(1)[0];
         quorumHash = readHash();
+        if (getVersion() >= INDEXED_QUORUM_VERSION) {
+            quorumIndex = readUint16();
+        } else {
+            quorumIndex = 0;
+        }
```

Comment:
this change will read the new format of quorums in Core 18.  Without this, the masternode list synchronization will fail.